app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -721,7 +730,29 @@ fun SettingsDialog(
                 text = stringResource(R.string.map_screen_mode_switch_label),
                 style = MaterialTheme.typography.bodyLarge)
           }
-
+          HorizontalDivider(
+              thickness = 1.dp,
+              modifier = Modifier.fillMaxWidth(),
+              color = MaterialTheme.colorScheme.onSurface.copy(alpha = 0.12f))
+          // Switch to Online/Offline Mode
+          Row(verticalAlignment = Alignment.CenterVertically, modifier = Modifier.fillMaxWidth()) {
+            Box(modifier = Modifier.width(64.dp), contentAlignment = Alignment.CenterStart) {
+              Switch(
```

Comment:
Can you please add a `contentDescription` for accessibility purposes ?

Reply:
For an image, ok if the user wants to know what the image represents, 
For an icon, ok if the user wants to know what does this icon represents.
But for a switch, the user will just see what the switch does with the text.
The text next to it acts as a content description, so no noeed.